cns/client/client_test.go

```diff
@@ -1708,16 +1715,18 @@ func TestReleaseIPAddress(t *testing.T) {
 	tests := []struct {
 		name     string
 		ctx      context.Context
-		ipconfig cns.IPConfigRequest
+		ipconfig cns.IPConfigsRequest
 		mockdo   *mockdo
 		routes   map[string]url.URL
 		wantErr  bool
 	}{
 		{
 			name: "happy case",
 			ctx:  context.TODO(),
-			ipconfig: cns.IPConfigRequest{
-				DesiredIPAddress: "testipaddress",
+			ipconfig: cns.IPConfigsRequest{
```

Comment:
this is missing test coverage where you would have multiple IPs to release.
Should add a test where if second IP release fails, CNS returns error to the client.